modal_proto/api.proto

```diff
@@ -979,6 +980,7 @@ message TaskLogsBatch {
   string function_id = 11;
   string input_id = 12;
   string image_id = 13;  // Used for image logs
+  bool eof = 14;
```

Comment:
is this one needed?

Reply:
Yeah, the worker needs to send a logs batch with `eof` (see https://github.com/modal-labs/modal/pull/6040)